examples/evp/1d_MHD_KH_left_eigenvectors/MHD_KH_EVP.py

```diff
@@ -0,0 +1,103 @@
+"""
+Runs an eigenvalue problem for the KH instability of a U = tanh(z) shear layer in MHD
+for purposes of testing left eigenvectors
+
+TODO: Remove this script before merging with main. This is just meant for testing.
```

Comment:
Do you want to keep this in? If so, I think it needs a little cleanup (particularly the comments). Otherwise it could also be reimplemented as a test.

Reply:
I think it's helpful as a demonstration of the biorthogonality relations, but seems like too minor/redundant of a thing to leave as an example script. I like the idea of implementing it as a test, but I'm not sure what that entails or how best to do that. Maybe we can chat further about this via email or zoom.

Reply:
@afraser3 we can chat about this on Thursday if you'd like...

Reply:
Evan walked me through a little bit of how to implement tests. I'll give it a shot tomorrow and see what happens. If it doesn't work out, I think it's reasonable to cut this from the PR.

Reply:
Properly testing bi-orthogonality is just a little more finicky than I realized, I think because of sorting left vs right eigenmodes and the inf/nans you get when doing dense solves. Is it fine to just cut the example altogether, or is that bad practice?

Reply:
I think it's ok, we can always pull this example out later if needed. Thanks!